.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: node_js
+
+sudo: false
+
+cache:
+    yarn: true
+    directories:
+        - node_modules
+
+env:
+    global:
+        - NODE_ENV=test
+        - TZ=Europe/Paris
+
+node_js:
+    - "8.10"
```

Comment:
Can't we pass to node 10 ?

Reply:
See https://github.com/20minutes/serverless-github-check/pull/1#discussion_r232693215